Sources/GRPCInProcessTransport/InProcessTransport.swift

```diff
@@ -16,24 +16,16 @@
 
 public import GRPCCore
 
-public enum InProcessTransport {
-  /// Returns a pair containing an ``InProcessServerTransport`` and an ``InProcessClientTransport``.
-  ///
-  /// This function is purely for convenience and does no more than constructing a server transport
-  /// and a client using that server transport.
+@available(macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0, *)
+public struct InProcessTransport: Sendable {
+  public let server = Self.Server()
```

Comment:
nit: initialize this in the `init`

```suggestion
  public let server: Self.Server
```